docs/CONTRIBUTING.md

```diff
@@ -25,15 +25,58 @@ We strive to maintain a friendly and inclusive community. We have not yet establ
 
 ## Development Setup
 
-To set up your development environment for firewatch, please follow these steps:
-
 1. Fork the repository on GitHub.
 2. Clone your forked repository to your local machine.
 3. Navigate to the project root: `cd firewatch`
 4. Install the necessary dependencies: `make dev-environment`
 5. Run pre-commit hooks to ensure code quality: `make pre-commit`
 6. Execute tests to ensure everything is working correctly: `tox` or `make test`
 
+## Testing Changes
+
+### Container
+
+   1. Find a failed prow job you would like to test against.
+      - **Example**: [periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws #1696039978221441024](https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-ci-openshift-pipelines-release-tests-release-v1.11-openshift-pipelines-ocp4.14-lp-interop-openshift-pipelines-interop-aws/1696039978221441024)
+   2. Fill out `firewatch/development/env.list`
+      - Use the ["Defining Environment Variables"](#defining-environment-variables) section to help
+   3. From the root of this repository, run `make docker-build-test` to execute `firewatch report` using the values provided above.
+      - You can also run `make docker-build-run` if you would like a bash terminal for the new build.
```

Comment:
Shouldn't these make commands be like `make build-test` and `make build-run` without `docker-*` looking at Makefile?

Reply:
I think it's better to be specific when we can, all is done via container so docker is the right name, But now that we can use also podman the `docker` needs to be changed
maybe container-build-tes and so on......

Reply:
@chetna14manku yes that was incorrect. I have updated the commands to be `container-` in both the makefile and the documentation as @myakove requested.